examples/address_generator.py

```diff
@@ -16,8 +16,8 @@
 from six import binary_type, moves as compat, text_type
 
 
-def main(uri, index, count):
-  # type: (Text, int, Optional[int], bool) -> None
+def main(uri, index, count, security, checksum):
+  # type: (Text, int, Optional[int], Optional[int], Optional[bool]) -> None
```

Comment:
`Optional[bool]` means that the value can be either a boolean or `None` ([PEP-484](https://www.python.org/dev/peps/pep-0484/#union-types)).

In this case, though, `None` is not an expected value for `checksum`, so we don't need the `Optional` part.